config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
   namespace :api do
-    namespace :v0 do
-      resources :pings, only: [:index], constraints: { format: 'json' }
-    end
+    get 'products/index'
```

Comment:
remove this please, we don't declare routes like this

Reply:
removed this route declaration and extra namespace that contained it